File: android/PetSitterISI/app/src/main/java/com/example/petsitterisi/entitees/Favoris.java
package com.example.petsitterisi.entitees;

public class Favoris {

    private int id;
    private int idPetSitter;
    private int idProprietaire;

    public Favoris(int id, int idPetSitter, int idProprietaire) {
        this.id = id;
        this.idPetSitter = idPetSitter;
        this.idProprietaire = idProprietaire;
    }

    public int getId() {
        return id;
    }

    public int getIdPetSitter() {
        return idPetSitter;
    }

    public int getIdProprietaire() {
        return idProprietaire;
    }
}

File: backend/src/models/favoris.js
class Favoris {
    constructor(id, id_petsitter, id_proprietaire){
        this._id = id
        this._id_petsitter = id_petsitter
        this._id_proprietaire = id
    }
    /* ######################################### GETTER #########################################*/

    get id(){
        return this._id
    }
    get id_petsitter(){
        return this._id_petsitter
    }
    get id_proprietaire(){
        return this._id_proprietaire
    }

    /* ######################################### GETTER #########################################*/

    set id(id){
        this._id = id
    }
    set id_petsitter(id_petsitter){
        this._id_petsitter = id_petsitter
    }
    set id_proprietaire(id_proprietaire){
        this._id_proprietaire = id_proprietaire
    }

}

module.exports = Favoris
